tools/azure-pipelines/build-python-wheels.yml

```diff
@@ -0,0 +1,82 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+jobs:
+  - job: compile_${{parameters.stage_name}}
+    pool:
+      vmImage: 'ubuntu-16.04'
+    container: flink-build-container
+    timeoutInMinutes: 240
+    cancelTimeoutInMinutes: 1
+    workspace:
+      clean: all # this cleans the entire workspace directory before running a new job
+      # It is necessary because the custom build machines are reused for tests.
+      # See also https://docs.microsoft.com/en-us/azure/devops/pipelines/process/phases?view=azure-devops&tabs=yaml#workspace
+
+    steps:
+      # The cache task is persisting the .m2 directory between builds, so that
+      # we do not have to re-download all dependencies from maven central for
+      # each build. The hope is that downloading the cache is faster than
+      # all dependencies individually.
+      # In this configuration, we use a hash over all committed (not generated) .pom files
+      # as a key for the build cache (CACHE_KEY). If we have a cache miss on the hash
+      # (usually because a pom file has changed), we'll fall back to a key without
+      # the pom files (CACHE_FALLBACK_KEY).
+      # Offical documentation of the Cache task: https://docs.microsoft.com/en-us/azure/devops/pipelines/caching/?view=azure-devops
```

Comment:
I don't think we need to copy that comment over as well

Reply:
Yes. I will remove the comment.